### PR TITLE
Fat 5569 create referenceforunits

### PR DESCRIPTION
**Pull request by Activict**

### CODE REVIEW

File: StoreApp.BLL/Services/UnitService.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace StoreApp.BLL.Services
+{
+    public class UnitService : IUnitService
+    {
+        private IUnitOfWork DataBase { get; set; }
+        private IMapper config;
```
Comment (AleksandroBoko) [inline StoreApp.BLL/Services/UnitService.cs]: Look at the style of declaration. I propose to follow one common approach

File: StoreApp.BLL/Services/UnitService.cs [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+        public void Delete(int id)
+        {
+            DataBase.Units.Delete(id);
+            DataBase.Save();
+        }
+
+        public void Detach(UnitDTO unit)
+        {
+            throw new NotImplementedException();
```
Comment (AleksandroBoko) [inline StoreApp.BLL/Services/UnitService.cs:54]: You did not implement this method

File: StoreApp/Views/Units/DeleteUnit.cshtml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            <div class="col-md-offset-2 col-md-10">
+                <input type="submit" value="Delete" class="btn btn-danger" />
+            </div>
+        </div>
+    }
+    else
+    {
+        <div class="form-group">
+            <div class="col-md-offset-2 col-md-10">
+                If you wont to delete @Model.Name you must to delete @ViewBag.CountProducts product(s)
```
Comment (AleksandroBoko) [inline StoreApp/Views/Units/DeleteUnit.cshtml]: Grammar mistake 
**If you want to**
**you must delete**

File: StoreApp/Views/Units/DetailsUnit.cshtml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    <hr />
+    <dl class="dl-horizontal">
+        <dt>
+            @Html.DisplayNameFor(model => model.Name)
+        </dt>
+
+        <dd>
+            @Html.DisplayFor(model => model.Name)
+        </dd>
+
```
Comment (AleksandroBoko) [inline StoreApp/Views/Units/DetailsUnit.cshtml:18]: Empty space which can be removed

File: StoreApp.BLL/Services/UnitService.cs (hunk 2 above)
Comment (Activict) [reply StoreApp.BLL/Services/UnitService.cs:54]: i will implement it after merge

File: StoreApp/Views/Units/DetailsUnit.cshtml (hunk 4 above)
Comment (Activict) [reply StoreApp/Views/Units/DetailsUnit.cshtml:18]: fixed

File: StoreApp/Views/Units/DeleteUnit.cshtml (hunk 3 above)
Comment (Activict) [reply StoreApp/Views/Units/DeleteUnit.cshtml]: fixed

File: StoreApp.BLL/Services/UnitService.cs (hunk 1 above)
Comment (Activict) [reply StoreApp.BLL/Services/UnitService.cs]: fixed

**Pull request merged**